### PR TITLE
updating donate page spacing & curly quotes

### PR DESCRIPTION
**Pull request by ryestew**

### CODE REVIEW

File: views/donate.jade [hunk 1]
```diff
@@ -49,50 +50,48 @@ block content
...
 
 
   div.main-tutorial.inner-page.inner-tutorial
-    div.container
+    div.container  
       div.row
         div.col-md-12
           include:md content/donate.md
-          h3
-            a(name="watch") Here's how to watch (and play with) your unicorns
```
Comment (alexvandesande) [inline views/donate.jade]: Why are we making this all caps?

File: dist/cli.html [hunk 2]
```diff
@@ -85,8 +85,6 @@
 
 <h4>Install on Windows</h4>
 
-<p>There is an <a href="http://www.ethdocs.org/en/latest/ethereum-clients/cpp-ethereum/installing-binaries/windows-installer.html">Installer for Windows</a>.</p>
```
Comment (alexvandesande) [inline dist/cli.html]: Should this be here or not? It is removed on the cli.md

**Comment by ryestew** [comment]
I think the caps are a mistake (an artifact of copying text) and should be lower case as they were.

I don't know why the cli.html was changed. It should not be changed by my PR. ( and yet it was)

-Rob

**Comment by rados-io** [comment]
Fairly positive you should make changes in sources and not in `dist` folder, otherwise they will be overwritten during deploy

**Pull request merged**